src/main/java/trackerbot/gui/DialogBox.java

```diff
@@ -0,0 +1,94 @@
+package trackerbot.gui;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.geometry.Pos;
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+import javafx.scene.shape.Circle;
+
+/**
+ * Represents a dialog box consisting of an ImageView to represent the speaker's face and a label
+ * containing text from the speaker.
+ * <p>Skeleton code from the JavaFX tutorial provided.</p>
+ *
+ * @version Level-10
+ */
+public class DialogBox extends HBox {
+    /** Label for the DialogBox. */
+    @FXML
+    private Label dialog;
+
+    /** Avatar image for the DialogBox. */
+    @FXML
+    private ImageView displayPicture;
+
+    /**
+     * Constructs the DialogBox controller object.
+     * @param text The description to display in the box.
+     * @param img The image of the avatar.
+     */
+    private DialogBox(String text, Image img) {
+        try {
+            FXMLLoader fxmlLoader = new FXMLLoader(MainWindow.class.getResource("/view/DialogBox.fxml"));
+            fxmlLoader.setController(this);
+            fxmlLoader.setRoot(this);
+            fxmlLoader.load();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        dialog.setText(text);
+        displayPicture.setImage(img);
+
+        Circle circleClip = new Circle(
```

Comment:
I like that your statements here have indentation of 8 spaces and that the lines are split appropriately.